department-of-veterans-affairs/abd-vro
# Chengjie8/heap out of memory fix

## What was the problem?
have issues connect to kafka broker due to the ssl configs from application yaml files not being defined correctly

Associated tickets or Slack threads:
- found during latest testing after heap out of memory issue was resolved

## How does this fix it?[^1]
Moving ssl config under consumer in yaml file

## How to test this PR
- can check k8s svc-bie-kafka pod log and verify if there's no connection exceptions in the log 


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
